source/help/00README.md

```diff
@@ -181,7 +182,7 @@ Here the `compiler: latex` is actually shorthand for `compiler: latex+dvips_ps2p
 
 Furthermore, note that we have not included the `usage: toplevel` line for `my_super_paper.tex`, since this will be automatically added.
 
-In the next example, let us explore some options that are rarely used but supported in the original 00README: the `landscape`, `keep_comments`, and `fontmaps` entry. The example uses a toplevel tex file `my_super_paper.tex`, but requests it be compiled with `tex` (not `latex`), and the postprocessing makes sure that the generated `dvi` file is converted in landscape mode, and keeps comments (by using `-K0`). Furthermore, `dvips` should use additional font map files `special1-fontmap.map` and `special2-fontmap.map`.
+In the next example, let us explore some options that are rarely used but supported in the original `00README`: the `landscape`, `keep_comments`, and `fontmaps` entry. The example uses a toplevel tex file `my_super_paper.tex`, but requests it be compiled with `tex` (not `latex`), and the postprocessing makes sure that the generated `dvi` file is converted in landscape mode, and keeps comments (by using `-K0`). Furthermore, `dvips` should use additional font map files `special1-fontmap.map` and `special2-fontmap.map`.
```

Comment:
Suggestion: In second sentence change "tex file" to "TeX file" to match other usage on this page.